gabrielgomesml/training_project_server
# Feature/Socket

## What I did

- Add Socket and create the events for Discussion Chat

## How to test

- First of all make sure you have the docker installed on your machine
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- Run the command `docker-compose up -d` to create a docker container with the postgres database
- After that run the command `yarn typeorm migration:run` or `yarn migration` to update the database with migrations and `yarn start` for run the project
